pom.xml

```diff
@@ -21,13 +21,13 @@
 
     <properties>
         <!-- Last Jenkins LTS release -->
-        <jenkins.version>1.580.1</jenkins.version>
-        <jenkins-test-harness.version>1.580.1</jenkins-test-harness.version>
```

Comment:
The reason for upgrading? Most similar plugins stick with the older version.

Reply:
Apparently, I don't need all the pipeline step dependencies that I thought I did. These versions can stay as they are.